fe2-android/app/src/test/framework/common/java/com/github/dedis/popstellar/testutils/pages/lao/digitalcash/HistoryPageObject.java

```diff
@@ -14,4 +14,44 @@ private HistoryPageObject() {
   public static int fragmentDigitalCashHistoryId() {
     return R.id.fragment_digital_cash_history;
   }
+
+
+  // create a function for all these elements :     // Click on the first transaction
+    //    onView(withId(HistoryPageObject.transactionCardView()))
+    //      .perform(ViewActions.click())
+    //
+    //    // Check if the transaction details are displayed
+    //    onView(withId(HistoryPageObject.transactionProvenanceTitle())
+    //    )
+    //      .check(matches(isDisplayed()))
+    //    onView(withId(HistoryPageObject.transactionProvenanceValue())
+    //    )
+    //      .check(matches(isDisplayed()))
+    //    onView(withId(HistoryPageObject.transactionIdValue())
+    //    )
```

Comment:
Do you need these comments to be here?

Reply:
just some github copilot prompts I forgot to remove, nice catch!